README.md

```diff
@@ -316,6 +317,12 @@ docker run -p 8080:80 \
     bgruening/galaxy-stable
 ```
 
+## On-demand reference data with CVMFS <a name="cvmfs" /> [[toc]](#toc)
+By default, Galaxy instances launched with this image will have on-demand access to approximately 3TB of reference genomes and indexes. These are the same reference data available on the main Galaxy server. This is achieved by connecting to Galaxy's CernVM filesystem (CVMFS) server at `data.galaxyproject.org`. The CVMFS capability doesn't add to the size of the Docker image, but when running, CVMFS maintains a cache to keep the most recently used data on the local disk.
+
```

Comment:
Might want to reword this slightly, for example, "This is achieved by connecting to Galaxy's CernVM filesystem (CVMFS) `data.galaxyproject.org` repository, which is geographically distributed among numerous servers.

Reply:
Fixed here: https://github.com/bgruening/docker-galaxy-stable/commit/4ad9466daf07bc8e5f1ab7f2ca78bde3ca4adffe